code/modules/research/designs/medical_designs.dm

```diff
@@ -399,6 +399,21 @@
 	build_path = /obj/item/organ/internal/cyberimp/arm/razorwire
 	category = list("Medical")
 
+/datum/design/cyberimp_sensory_enhancer
+	name = "Qani-Laaca Sensory Computer Implant"
+	desc = "An experimental implant replacing the spine of organics. When activated, it can give a temporary boost to mental processing speed, \
+		Which many users perceive as a slowing of time and quickening of their ability to act. Due to its nature, it is incompatible with \
+		system that heavily influence the user's nervous system, like the central nervous system rebooter. \
+		As a bonus effect, you are immune to the burst of heart damage that comes at the end of twitch usage, as the computer is able to regulate \
+		your heart's rhythm back to normal after its use."
```

Comment:
```suggestion
	desc = "An experimental implant replacing the spine of organics. When activated, it can give a temporary boost to mental processing speed, \
		Which many users perceive as a slowing of time and quickening of their ability to act. Due to its nature, it is incompatible with \
		systems that heavily influence the user's nervous system, like the central nervous system rebooter. \
		As a bonus effect, you are immune to the burst of heart damage that comes at the end of twitch usage, as the computer is able to regulate \
		your heart's rhythm back to normal after its use."
```